src/game/screen_transition.c

```diff
@@ -24,7 +24,7 @@ void *sTextureTransitionID[] = {
 };
 
 
-s32 set_and_reset_transition_fade_timer(u8 transTime) {
+s32 set_and_reset_transition_timers(u8 transTime) {
```

Comment:
This is only resetting one timer.

Reply:
It also resets the angle, how about set_and_reset_transitions_values?

Reply:
I think old name was probably fine tbh